client/dashboard/sites/site-preview/index.tsx

```diff
@@ -18,6 +18,12 @@ export default function SitePreview( {
 	const secureUrl = url.replace( /^http:\/\//, 'https://' );
 	return (
 		<iframe
+			// Enabling sandbox disables most features, such as autoplay,
```

Comment:
Perhaps we should remove `autoplay` as one of the examples in this comment though.